docs/satellitev2.rst

```diff
@@ -187,13 +187,13 @@ Detect
     * :code:`test_url` : String
         The domain being queried.
     * :code:`response` : JSON object
-        The resolver's returned answer IPs for the queried domain are the keys. Each answer IP is mapped to an array of its tags that matched the control tags - if the IP is in the control set, "ip" is appended and if the IP has no tags, "no_tags" is appended. Also has an :code:`rcode` field mapping to a list of response codes for the trials.
+        The resolver's returned answer IPs for the queried domain are the keys. Each answer IP is mapped to an array of its tags that matched the control tags - if the IP is in the control set, "ip" is appended and if the IP has no tags, "no_tags" is appended. The :code:`rcode` field maps to a list of response codes for the trials. The :code:`error` field maps to a list of response error messages in case of connection errors.
```

Comment:
Can we please add that the `rcode` field is set to -1 in case of any connection errors.

Reply:
We mentioned in the description for `query` module: Response code mapping to success (0) or errors (-1 for connection error, > 0 for errors specified in RFC 2929). Maybe we can omit this in the detect module to avoid repetition?